systest/plugin/plugin_test.go

```diff
@@ -332,5 +305,47 @@ func TestPlugins(t *testing.T) {
 				]}`,
 			},
 		},
-	)
+	},
+}
+
+func (psuite *PluginTestSuite) TestPlugins() {
+	t := psuite.T()
+
+	if testing.Short() {
+		t.Skip("skipping test in short mode")
+	}
+
+	for i := 0; i < len(testInp); i++ {
+		psuite.dataSetNdx = i
+		psuite.Run(fmt.Sprintf("test case %d", i+1), psuite.pluginFn)
```

Comment:
See whether we can switch to using t.Run or something like we discussed for the other PR